torch_geometric/explain/explainer.py

```diff
@@ -203,3 +203,55 @@ def _threshold(self, explanation: Explanation) -> Explanation:
             explanation[key] = mask
 
         return explanation
+
+    def visualize_feature_importance(
+        self,
+        explanation: Explanation,
+        feat_labels: Optional[List[str]] = None,
+        top_k: int = 20,
+    ):
+        r"""Creates a bar plot of the node features importance given
+        a :attr:`node_feat_mask`
+
+        Args:
+            explanation (Explanation): The explanation object.
+            feat_labels (List[str], optional): Optional labels for features.
+            top_k (int, optional): Top K features to plot.
+                (default: :obj:`20`)
```

Comment:
Can we set default value to `None`. And if this is `None` plot all features.

Reply:
I think we should have a `top_k`, in the example case, there are >1400 features and the visualisation breaks when we try to plot all of them.

Another option would be to plot all feature importance if the number of features is below a default `k`. Else, plot importance of the the top 20 feats. 


<img width="828" alt="Screenshot 2022-12-07 at 17 08 58" src="https://user-images.githubusercontent.com/36218572/206244765-5acf4ffd-a847-4072-900b-97828dd27722.png">

Reply:
I agree that we should retain the `top_k` argument for reasons you just pointed out. But `top_k` can be optional, and if its not passed we just plot all features. With that the users does need to worry about `top_k` if they have few features.